src/nft/components/profile/list/SelectMarketplacesDropdown.tsx

```diff
@@ -0,0 +1,172 @@
+import { useOnClickOutside } from 'hooks/useOnClickOutside'
+import { Column, Row } from 'nft/components/Flex'
+import { ChevronUpIcon } from 'nft/components/icons'
+import { Checkbox } from 'nft/components/layout/Checkbox'
+import { buttonTextMedium, caption } from 'nft/css/common.css'
+import { ListingMarket } from 'nft/types'
+import { ListingMarkets } from 'nft/utils/listNfts'
+import { Dispatch, FormEvent, useMemo, useReducer, useRef } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+import { Z_INDEX } from 'theme/zIndex'
+
+const HeaderButtonWrap = styled(Row)`
+  padding: 12px;
+  border-radius: 12px;
+  width: 220px;
+  justify-content: space-between;
+  background: ${({ theme }) => theme.backgroundModule};
+  cursor: pointer;
+  &:hover {
+    background-color: ${({ theme }) => theme.backgroundInteractive};
+  }
+`
+
+const HeaderButtonContentWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+
+const MarketIcon = styled.img<{ index: number; totalSelected: number }>`
+  height: 20px;
+  width: 20px;
+  margin-right: 8px;
+  border: 1px solid;
```

Comment:
nit: can just put the color here

Reply:
I personally prefer having the conditional as atomic as possible

Reply:
@cbachmeier there is no conditional in this case?